src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -171,11 +172,13 @@ fi
         pythonBuildExitCode=${PIPESTATUS[0]}
         ELAPSED_TIME=$(($SECONDS - $START_TIME))
         echo "Done in $ELAPSED_TIME sec(s)."
+        set -e
         if [[ $pythonBuildExitCode != 0 ]]
         then
             LogWarning "${StdWarning} | Exit code: {pythonBuildExitCode} | Please review message | ${moreInformation}"
             exit $pythonBuildExitCode
         fi
+        set -e
```

Comment:
Nit: do you need this one given that you already added one line 175?

Reply:
No, thanks for catching this :)